packages/react-native-windows-init/src/Cli.ts

```diff
@@ -0,0 +1,291 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as yargs from 'yargs';
+import * as fs from 'fs';
+import * as semver from 'semver';
+import {execSync} from 'child_process';
+import * as validUrl from 'valid-url';
+import * as prompts from 'prompts';
+import * as findUp from 'find-up';
+import * as chalk from 'chalk';
+
+const Registry = require('npm-registry');
+
+let npmConfReg = execSync('npm config get registry')
```

Comment:
nit: the rest of this file is using `const`. This signals usages of `let` to be cases where we mutate the variable, which isn't true here. Could we change these to `const`?